File: solutions/_1758_minimum_changes_to_make_alternating_binary_string/solution.cpp
#include <string>
using namespace std;

class Solution
{
public:
    int minOperations(string s)
    {
        int n = s.size();
        int count1 = 0, count2 = 0;
        for (int i = 0; i < n; i++)
        {
            if (s[i] - '0' == i % 2)
                count1++;
            else
                count2++;
        }
        return min(count1, count2);
    }
};

/*
https://leetcode.com/problems/minimum-changes-to-make-alternating-binary-string/
*/

File: solutions/_1758_minimum_changes_to_make_alternating_binary_string/solution.py
class Solution:
    def minOperations(self, s: str) -> int:
        n = len(s)
        count1 = 0
        count2 = 0
        for i in range(n):
            if int(s[i]) == i % 2:
                count1 += 1
            else:
                count2 += 1
        return min(count1, count2)


# https://leetcode.com/problems/minimum-changes-to-make-alternating-binary-string/
